app/views/photos/new.html.erb

```diff
@@ -1,16 +1,35 @@
 <div class="page-header"><h1>Upload Photo</h1></div>
```

Comment:
We probably need a `document ready` because the dropzone feature doesn't show up until we refresh thepage